CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+ Contributing
+
+We love pull requests from everyone. Contributing is easy:
+
+* You can report bugs and request features using the [issues page][issues].
+
+[issues]: https://github.com/pydatagn/Community/issues
+
+* You can easily work on the code if you have a GitHub account ([get one][github]):
+
+[github]: https://github.com/join
+
+Fork the project, then clone the repo:
```

Comment:
Can you make this more specific to our repository

Reply:
How exactly do you see this Contributors' Hall of fame to be implemented, auto-population of the file or manual filling of this file?
Can you give more specific ideas for this plan.

Reply:
Yeah, will change the link to be specific to the repo.

Reply:
I already finished working on that. Made a basic contributors.md file. Feel free to make it better

Reply:
Oh cool.
Just checked it. Guess I will link this in the CONTRIBUTING file in place of assistance.